YAMS-LIB/Program.cs

```diff
@@ -2147,14 +2147,16 @@ void RotateTextureAndSaveToTexturePage(int rotation, UndertaleTexturePageItem te
 
         // Flashlight
         PrependGMLInCode(characterVarsCode, "global.flashlightLevel = 0;");
+        ReplaceGMLInCode(gmData.Code.ByName("gml_Script_ApplyLightPreset"), "global.darkness", "lightLevel");
```

Comment:
When i read this for the first time, i was surprised GML lets you declare a normal variable, and then later redeclare it as a temp variable